code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -387,6 +387,17 @@
 	max_ammo = 80
 	multiple_sprites = 1
 
+//Magazine type for the mech PK, you shouldn't see this
+
+/obj/item/ammo_magazine/lrifle/pk/mech
+	name = "LMG munitions box (.30 Rifle, Exosuit)"
+	icon_state = "pk_box"
+	matter = list()
+	spawn_blacklisted = TRUE
+	bad_type = /obj/item/ammo_magazine/lrifle/pk/mech
```

Comment:
The LMG Munitions box the mech uses is replace-able by attacking the weapon with it , im not sure how well that does work but it does work.  Did you make it just load this mag on the spot instead of replacing it inside the mech gun?

Reply:
Uh.. I'm not too sure actually, I did change what mag the gun used inside. What effect does replacing the lmg munitions box have? From playtesting the mech just made more ammo after it went empty.

Reply:
oh i don't remember it automatically replacing, its probably not even intended.

Reply:
hm.. In #5628 Shown does say it regenerates boxes for 500 energy a piece.

Reply:
oh didn't know it was intended,  never saw a code comment about it , all fine then.

Reply:
all good man